src/gpu-compute/scoreboard_check_stage.cc

```diff
@@ -196,6 +196,21 @@ ScoreboardCheckStage::ready(Wavefront *w, nonrdytype_e *rdyStatus,
             return false;
         }
     }
+
+    if (ii->isMFMA()) {
+        if (computeUnit.matrix_core_ready[w->simdId] <= curTick()) {
+            computeUnit.matrix_core_ready[w->simdId] =
+                curTick() +
+                computeUnit.cyclesToTicks(Cycles(
+                    computeUnit.mfma_scale *
+                    computeUnit.mfma_cycles[w->gfxVersion]
```

Comment:
Probably we need an assert somewhere in this code block (between lines 201 and 200?) that fails if MFMAs are not supported in the GFX version being used?

Reply:
done